pootle/apps/pootle_store/unit/results.py

```diff
@@ -88,7 +73,6 @@ def data(self):
                     'project_style': unit.project_style}
             units_list.append(
                 {'id': unit.id,
-                 'url': unit.translate_url,
```

Comment:
thanks for pointing this out - it was showing in coveralls as not being used, after i had made a change.

Ill add it back in.